velox/exec/tests/PartitionedOutputBufferManagerTest.cpp

```diff
@@ -311,6 +311,13 @@ class PartitionedOutputBufferManagerTest : public testing::Test {
     fetchedPages = nextSequence;
   }
 
+  void verifyOutputBufferUtilization(std::shared_ptr<Task> task) {
```

Comment:
```
s/EXPECT_GT/ASSERT_GT/
```
dittos
Also can we pass 0, 1 as expected value?
```
expectedExclusiveUtitlizationLowerBound
expectedExclusiveUtitlizationUpperBound
expectedBufferOverUtilization
```

In addition to this test, it might be better to have a dedicated test case to explicitly check
1. zero utilization on a fresh new output buffer
2. over utilization with >100 utilization and with producers
3. non-over utilization with > 100 utilization and without producers?
Thanks!